### ISSUE TITLE
Incorrect title of the post-> myth-screen-readers-dont-use-javascript/

### ISSUE THREAD

**Issue by virenv** [a11yproject/a11yproject.com#785]
Please review our Contributing guidelines:
https://github.com/a11yproject/a11yproject.com/blob/gh-pages/CONTRIBUTING.md#reporting-issues

## Summary
The title of the post **_["Myth screen reader dont use Javascript" is](https://a11yproject.com/posts/myth-screen-readers-dont-use-javascript/)_** not correct. Technically screen readers are don't communicate or use JavaScript. Also, JavaScript on webpages are not accessible it is the pure HTML elements which are accessible. Even if a developer wants to use JavaScript they can make visual changes to the page only by producing/altering html content of the page.

I think what we mean here is that Screen readers don't interact with the web pages which rely on JavaScript. Which I still find incorrect. It "Myth: Screen Readers don't interact with JavaScript rich web pages"

### Expected result
Title of the post should be: "Myth: Screen Readers don't interact with JavaScript rich web pages"

### Actual result
Myth: Screen readers don't use JavaScript

**Comment by ericwbailey** [comment]
Assistive technology reads from the Accessibility Tree, which is modified by things like JavaScript. Screen readers can take action on items that are generated via JS, and that also triggers JS behavior. 

Additionally, some AT can read and operate non-semantic HTML that has had its behavior modified by ARIA, and oftentimes this ARIA is applied by JS. It's not ideal, but providing this ability to provide enhancements for the limitations of HTML it is one of the reasons ARIA was authored in the first place.

To one of your points, it doesn't matter the level of JS present. I think your suggested title is an unnecessary and potentially misleading qualifier, however.

**Comment by virenv** [comment]
@ericwbailey : I do understand what the technical details are. I am a developer working on the Accessibility on Edge and Chromium browser. 

`Assistive technology reads from the Accessibility Tree, which is modified by things like JavaScript. Screen readers can take action on items that are generated via JS, and that also triggers JS behavior.`

As far as I know, Javascript cannot modify the Accessibility tree. Javascript modifies the HTML tree. This change triggers a change in Accessibility tree. Which is then reread by a screen reader, mostly because of a structure/property changed event that the Application (browser in this case) sends. So Javascript or anything that inserts an HTML element in the tree will result in an update in the Accessibility tree. So to be technically correct Screenreaders dont really care to know that Javascript is doing anything or not. In recent days I have not seen ScreenReaders parsing the HTML tree to get information. May be very specific Javascript based readers (if there are any) could do it.

`Additionally, some AT can read and operate non-semantic HTML that has had its behavior modified by ARIA, and oftentimes this ARIA is applied by JS. It's not ideal, but providing this ability to provide enhancements for the limitations of HTML it is one of the reasons ARIA was authored in the first place.`

AT's might used to work on reading HTML for IE. These days browser convert the ARIA tags in html and pass it in relevant form to the AT. For e.g. if an Aria label button is used. Browser will create an Accessility node which has a control type of button for that element. So now AT dont really have to care about how the ARIA tags as such.

I think the title I have suggested is very appropriate. If can provide a better suggestion if you like. The current title seems to be misleading to me. I am all open for changing my mind for a valid reason.

**Comment by ericwbailey** [comment]
What I am driving at is that your title suggests an unnecessary extra degree of complexity to the purpose of the article, which is to address the myth that AT users need an HTML-only experience. This stems from the old ([but still existing](http://www.mondaq.com/unitedstates/x/766354/IT+internet/DOT+Fines+Scandinavian+Airline+System+For+Violating+DOTs+Website+Accessibility+Requirements)) practice of providing a separate AT-only experience.

**Comment by virenv** [comment]
I am worried about the term "Screen Reader **_dont use_** JavaScript".  To best of my knowledge it is not correct and it is not representing what the article is about.

**Comment by scottaohara** [comment]
Suggestion:
"Myth screen reader users don't use Javascript"

based on the short prose of the article, this revised title aligns with the intent of the original piece (specifically noting the WebAIM stat that indicates 98% of users have JS enabled).  This will still serve to debunk the myth that while in the past screen readers didn't work well with pages that were dynamically updated with JavaScript, this has not been an actual issue for a long time now, and for anyone who still thinks that a JavaScript heavy accessible user interface can't also be heavily dependent on JS, they are incorrect.

Is this change suitable?

**Comment by virenv** [comment]
@scottaohara : Sounds good to me.

**Comment by scottaohara** [comment]
closed by #786 

thank you
